api/models/Collaborator.js

```diff
@@ -110,6 +110,11 @@ module.exports = {
         var query = "SELECT coll.name, Sum(direct_downloads) as total From Packages pack INNER JOIN PackageVersions versions ON pack.latest_version_id=versions.id INNER JOIN Collaborators coll ON versions.maintainer_id = coll.id INNER JOIN DownloadStatistics downloads ON pack.name=downloads.package_name WHERE "+
           "downloads.date >= current_date() - interval '1' month group by coll.name order by total desc limit 0,10";
         return sequelize.query(query, {type: sequelize.QueryTypes.SELECT});
+      },
+
+      quickSearch: function(pattern){
+        var query = "SELECT distinct name  FROM Collaborators WHERE  name LIKE '%"+pattern+"%' limit 0,4"
+        return sequelize.query(query, {type: sequelize.QueryTypes.SELECT});
```

Comment:
Would be nice if they are ordered by aggregated download of the last month. 
You think you can try that ?